### DISCUSSION TITLE
different wiring

### DISCUSSION THREAD

**Discussion by Piman74** [tagyoureit/nodejs-poolController#569] [RS485]
Hi guys
I bought an INTELIFLO VSF pump and I cannot communicate with it on the RS485 port.
the pins of the RS485 connection are different, it is not the usual ports that are wired
The pump socket only has 2 wire pins 7 and 4 instead of 6 and 7.
I bought a first RS485 adapter and I have no communication (https://www.amazon.com/dp/B07SD65BVF/?coliid=I2ZXMUX22W8KIE&colid=151RLMN7IIZ9X&ref_=lv_ov_lig_dp_it&th=1)
I bought another model and the same no communication (https://www.amazon.com/gp/product/B081MB6PN2/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1)

The 2 adapters are well recognized on Windows, I put well in 9600/8/1stop no control

on the doc of the pump there is written that the communication is in RS485.

where is my mistake?s

**Comment by rstrouse** [comment]
Did you add the pump to the configuration?  Nixie needs to know that there is a pump there.  It will not try to talk to a device that it does not know exists.

Did this pump come with a comms cable.  If it did the pin mapping should already be identified.  I believe all of the IntelliFlo comms cables are the same no matter whether it is a VS or VSF pump.   Is it possible that you have a SuperFlo VST?
